backend/app/api/routers/volunteer.py

```diff
@@ -0,0 +1,46 @@
+from http import HTTPStatus
+from typing import Optional
+
+from fastapi import APIRouter
+
+from app.api.exceptions import EntityNotFound
+from app.api.schemas import Filter, Volunteer, VolunteerCreate
+from app.api.services import VolunteerService
+
+router = APIRouter()
+
+
+@router.post('/volunteers', response_model=Volunteer, status_code=HTTPStatus.CREATED)
+async def create_volunteer(volunteer: VolunteerCreate) -> Volunteer:
+    volunteer_new = await VolunteerService.create(volunteer)
+
+    return volunteer_new
+
+
+@router.get('/volunteers', response_model=list[Volunteer])
+async def find_all_volunteers(flt: Optional[Filter] = None) -> list[Volunteer]:
+    volunteers = await VolunteerService.find_all(flt)
+
+    return volunteers
+
+
+@router.get('/volunteers/{volunteer_tg_id}', response_model=Volunteer)
+async def find_volunteer(volunteer_tg_id: int) -> Volunteer:
+    volunteer = await VolunteerService.find_by_tg_id(volunteer_tg_id)
+
+    if not volunteer:
```

Comment:
The way it is written now, one could think that some volunteers have the `__bool__` magic method overloaded. Let's make this more explicit, e.g. `if volunteer is None`